bin/verilator

```diff
@@ -320,6 +321,7 @@ detailed descriptions in L</"VERILATION ARGUMENTS"> for more information.
      +libext+<ext>+[ext]...     Extensions for finding modules
     --lint-only                 Lint, but do not make output
     --max-num-width <value>     Maximum number width (default: 64K)
+    --max-proc <value>          Maximum number of CPUs to be used for build
```

Comment:
-j is free, let's use that unless you see some problem.  Please support -j no_number_here -next_argument, along with -j number_might_be_zero -next_argumnet, same as what make does.

Please also add -MAKEFLAGS passed down to the make tool.